ui/src/features/Editor/components/OverlayUI/components/DebugActionBar/components/DebugActiveRunsPopover.tsx

```diff
@@ -0,0 +1,112 @@
+import { BinocularsIcon } from "@phosphor-icons/react";
+
+import {
+  IconButton,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  ScrollArea,
+} from "@flow/components";
+import { useT } from "@flow/lib/i18n";
+import type { UserDebug } from "@flow/types";
+
+import CollaborationCard from "../../Homebar/components/CollaborationActionBar/components/CollaborationCard";
```

Comment:
CollaborationCard should move to a more appropriate directory if its being used outside of the Homebar now 🙏🏻